grafana-pyroscope.yaml

```diff
@@ -0,0 +1,105 @@
+package:
+  name: grafana-pyroscope
+  version: 1.13.1
+  epoch: 0
+  description: Continuous Profiling Platform. Debug performance issues down to a single line of code
+  copyright:
+    - license: AGPL-3.0-only
+
+environment:
+  contents:
+    packages:
+      - busybox
```

Comment:
In here, go,busybox,ca-certificates-bundle will be pulled in via go/build. Is python3 required for frontend part? (thinking)

Reply:
yep python3 is required for frontend build especially for node-gyp